ee/modules/140-user-authz/images/webhook/web/hook/handler.go

```diff
@@ -74,8 +74,7 @@ func (h *Handler) ServeHTTP(w http.ResponseWriter, r *http.Request) {
 
 	var request WebhookRequest
 	if err := json.NewDecoder(r.Body).Decode(&request); err != nil {
-		// this case is exceptional
-		h.logger.Fatalf("cannot unmarshal kubernetes request: %v", err)
```

Comment:
Do we need to authorize an empty request? It seems like, at this point, we should return an error to a user with the `bad request` code.

Reply:
Done
![image](https://github.com/deckhouse/deckhouse/assets/39344422/2f3c348d-e934-4303-afa2-7cc3153ac9ec)